### ISSUE TITLE
Default RPM repo file sets expiry time too low

### ISSUE THREAD

**Issue by cyqsimon** [crowdsecurity/crowdsec#1060]
This is an issue I posted over Gitter - so I'll simply paste it over:

> I think the default RPM `.repo` file's metadata expiry is set too short. Currently it's 5 minutes which is entirely unnecessary. For reference the default value for `dnf` is 48 hours.
It becomes annoying during maintenance when I'm installing, uninstalling, querying, searching, updating, etc. packages with `dnf`. Basically any `dnf` command would trigger a metadata refresh for crowdsec. And as anyone who uses RHEL can tell you, updating repo metadata is often irritatingly slow.
It's not a difficult fix. I simply commented out `metadata_expire=300` in `/etc/yum.repos.d/crowdsec_crowdsec.repo` and it reverts to the default 48h. But a more sensible default is always better IMO.

### tl;dr

Make the default something more sensible, like 12h, or just remove the line to use the default 48h.

**Comment by LaurenceJJones** [comment]
@sabban 

Do we have access to this via our settings in packagecloud?

**Comment by sabban** [comment]
I believe we don't.

**Comment by LaurenceJJones** [comment]
> Was this fixed? Or is this a "won't fix"?

Sorry it should of been placed under "wont fix" due to RPM time being out of our control within the installation script. If in the future we might have an external script for ourselves due to any/any requirements then we will have more control
